insights/parsers/tests/test_satellite_mongodb_storage_engine.py

```diff
@@ -0,0 +1,39 @@
+import pytest
+import doctest
+from insights.tests import context_wrap
+from insights.parsers import SkipException
+from insights.parsers import satellite_mongodb_storage_engine
+
+
+MONGO_PULP_STORAGE_ENGINE_OUTPUT1 = '''
```

Comment:
I met this failure in my test:
~~~
~]# /usr/bin/mongo pulp_database --eval 'db.serverStatus().storageEngine'
MongoDB shell version v3.4.9
connecting to: mongodb://127.0.0.1:27017/pulp_database
2020-02-13T23:19:57.750-0500 W NETWORK  [thread1] Failed to connect to 127.0.0.1:27017, in(checking socket for error after poll), reason: Connection refused
2020-02-13T23:19:57.751-0500 E QUERY    [thread1] Error: couldn't connect to server 127.0.0.1:27017, connection attempt failed :
connect@src/mongo/shell/mongo.js:237:13
@(connect):1:6
exception: connect failed
~~~

Please confirm if it's necessary for the Parser to raise a ParserException or SkipException for it.

Reply:
> I met this failure in my test:
> 
> ```
> ~]# /usr/bin/mongo pulp_database --eval 'db.serverStatus().storageEngine'
> MongoDB shell version v3.4.9
> connecting to: mongodb://127.0.0.1:27017/pulp_database
> 2020-02-13T23:19:57.750-0500 W NETWORK  [thread1] Failed to connect to 127.0.0.1:27017, in(checking socket for error after poll), reason: Connection refused
> 2020-02-13T23:19:57.751-0500 E QUERY    [thread1] Error: couldn't connect to server 127.0.0.1:27017, connection attempt failed :
> connect@src/mongo/shell/mongo.js:237:13
> @(connect):1:6
> exception: connect failed
> ```
> 
> Please confirm if it's necessary for the Parser to raise a ParserException or SkipException for it.

The satellite on your VM doesn't run, right? Since I just get "name" from the output, so in this case, it will fail to get the name, then raise SkipException. Let me add ParserException too if the name format isn't expected.

Reply:
Good